html/code/modules/roles/xardata/table_schema-def.xml

```diff
@@ -6,31 +6,31 @@
       <number size="10" unsigned="true" />
     </column>
     <column id="roles.name" name="name" required="true" >
-      <text size="255" charset="utf8"/>
+      <text size="255" charset=" utf8"/>
```

Comment:
Remove these spaces. I have adjusted the XSL to add a space.

Reply:
okay I will do it.